### ISSUE TITLE
INSERT float ARRAY with parameter not work.

### ISSUE THREAD

**Issue by jorise7** [cockroachdb/cockroach#19251]
```
db.Exec("CREATE TABLE foo (x INT ARRAY)")
db.Exec("INSERT INTO foo VALUES ($1)", "{1,2,3}")
```

it works.

```
db.Exec("CREATE TABLE foo (x FLOAT ARRAY)")
db.Exec("INSERT INTO foo VALUES ($1)", "{1.0,2.0,3.0}")
```

it didn’t work. i got this error:
pq: error in argument for $1: unsupported OID 2277 with format code formatText

cockroach 1.1 stable
golang / pq

**Comment by justinj** [comment]
Hey @jorise7, sorry about this! It is a problem on our end, there should be a fix ready for 1.1.1. Thanks for the report!

**Comment by jorise7** [comment]
thanks for your work.
